pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
```

Comment:
> I would suggest creating a property for the unclassified failures count that returns an integer based on this string

That part I've excerpted above, is test-side, correct?

I'll add a suitable ID to the UI, as I'm pretty confident we'll only have one location in the page that will ever display the count. I'll let you know when that's landed and available for testing. :)